src/js/components/CreateServiceModalServicePickerOption.js

```diff
@@ -0,0 +1,48 @@
+import {Hooks} from 'PluginSDK';
+import React from 'react';
+
+import CreateServiceModalServicePickerOptionContent from './CreateServiceModalServicePickerOptionContent';
+import CreateServiceModalServicePickerOptionImage from './CreateServiceModalServicePickerOptionImage';
+import Panel from './Panel';
+
+function getChildOfType(children, type) {
+  return React.Children.toArray(children).filter(
+    (child) => child.type === type
+  );
+}
+
+function CreateServiceModalServicePickerOption({children, onOptionSelect}) {
+  const columnClass = Hooks.applyFilter(
```

Comment:
This is done here: https://github.com/dcos/dcos-ui/pull/1782/commits/48bebb3d923489a834fc84886bbab7d20ace4718#diff-f1dca08b108e502cc34fdf15d8a8c291